internal/pkg/admission/validating-webhook.go

```diff
@@ -34,7 +34,7 @@ func (r *Reconciler) validatingWebhookConfiguration(
 
 	service := admissionregistrationv1.ServiceReference{
 		Namespace: r.DeploymentsNamespace,
-		Name:      constants.PolicyServerServiceName,
+		Name:      constants.PolicyServerServiceNamePrefix,
```

Comment:
The name of the service is wrong, should also include the name of this instance of PolicyServer